app/build.gradle

```diff
@@ -26,7 +26,7 @@ android {
         minSdkVersion 15
         targetSdkVersion 23
         versionCode 1
-        versionName "1.0"
+        versionName "3.0"
```

Comment:
The [current](https://play.google.com/store/apps/details?id=uk.co.es4b.tvarkau_vilniu&hl=lt) version is 2.1 We chose 3.0 because the app has significant differences from the previous version.